### ISSUE TITLE
Update Lasagne installation doc to new gpuarray backend

### ISSUE THREAD

**Issue by DreamFlasher** [Lasagne/Lasagne#897]
The documentation at http://lasagne.readthedocs.io/en/latest/user/installation.html to verify cuda and cudnn installation is outdated. It currently yields: `ValueError: You are tring to use the old GPU back-end. It was removed from Theano. Use device=cuda* now. See https://github.com/Theano/Theano/wiki/Converting-to-the-new-gpu-back-end%28gpuarray%29 for more information.`
Unfortunately the documentation at Theano does not contain the information to verify cuda and cudnn installation.

**Comment by f0k** [comment]
Yes, this documentation has not been updated for Theano 0.10+ yet, it will be updated at the latest when Lasagne drops support for Theano 0.8 (the last version of Theano that did not deprecate the old backend).
This being said, I can't find an equivalent for
```
THEANO_FLAGS=device=gpu python -c "import theano; print(theano.sandbox.cuda.device_properties(0))"
```
in the new backend. To verify whether cuDNN is available, run:
```
python -c "from theano.gpuarray.dnn import dnn_present as d; print(d() or d.msg)"
```

**Comment by nouiz** [comment]
If you use device=cuda for the new back-end, you have this printed during
the import:

Using cuDNN version 7001 on context None
Mapped name None to device cuda: Tesla P100-PCIE-12GB (0000:03:00.0)

So you see which version of cudnn is being used. It also show you the GPU
used (name and PCI-E bus in case you have multiple GPU in the same computer)

The installation is partialy tested during import. So you are sure we can
use cudnn and most of cuda. We do not test all libraries at import.

We didn't reimplement device_properties in the new back-end.

**Comment by f0k** [comment]
> If you use device=cuda for the new back-end, you have this printed during the import:

True, there's no need to query the device properties, simply importing Theano is enough to see whether the CUDA and cuDNN are used. The second command can still be useful for troubleshooting in case cuDNN is *not* being used.

I'm leaving this open since the documentation still needs to be updated later. (This requires more changes, it should include a section on libgpuarray/pygpu.)
